src/IotHub/IotHub.Test/ScenarioTests/IotHubRoutingTests.ps1

```diff
@@ -32,7 +32,7 @@ function Test-AzureRmIotHubRoutingLifecycle
 	$containerName2 = 'container2'
 	##############################################################
 
-	$Location = Get-Location "Microsoft.Devices" "IotHub" 
+	$Location = Get-Location "Microsoft.Devices" "IotHubs" "WEST US 2"
```

Comment:
Why change the `location`? Will this cause the change of test data?

Reply:
Message Enrichment feature is not available for some region, So added the location to fulfill test case requirement.